### ISSUE TITLE
iNat21 Pre-Training (fuzzy-fig)

### ISSUE THREAD

**Issue by samuelstevens** [samuelstevens/swin-transformer#11]
Pre-train swin-v2-base on iNat21 with default objective for 90 epochs at 192x192 with base learning rate of 1.25e-4.

**Comment by samuelstevens** [comment]
Done, and available at `s3://imageomics-models/fuzzy-fig-192-epoch89.pth`
